src/lava/magma/core/callback_fx.py

```diff
@@ -53,3 +54,23 @@ def post_run_callback(self,
                           board: NxBoard = None,
                           var_id_to_var_model_map: dict = None):
         pass
+
+
+class IterableCallBack(NxSdkCallbackFx):
+    """NxSDK callback function to execute iterable of function pointers
+    as pre and post run."""
+
+    def __init__(self,
+                 pre_run_fxs: Iterable = [],
+                 post_run_fxs: Iterable = []) -> None:
+        super().__init__()
+        self.pre_run_fxs = pre_run_fxs
+        self.post_run_fxs = post_run_fxs
+
+    def pre_run_callback(self, board: NxBoard, **_) -> None:
```

Comment:
Seems like codacy is complaining that you changed the type or name of the parameters. As you haven't change the numbers, as it seems to be saying.